frontend/app/mocks/power-platform-api.server.ts

```diff
@@ -77,8 +77,11 @@ export function getPowerPlatformApiMockHandlers() {
     /**
      * Handler for GET requests to retrieve letters details.
      */
-    http.get('https://api.example.com/users/:userId/letters', ({ params }) => {
-      const letterEntities = getLetterEntities(params.userId);
+    http.get('https://api.example.com/users/:userId/letters', ({ params, request }) => {
+      const url = new URL(request.url)
+      const sortParam = url.searchParams.get('sort') || 'asc'
+      const sort: 'asc' | 'desc' = sortParam === 'desc' ? 'desc' : 'asc';
```

Comment:
No need to explicitly type the variable.

```suggestion
      const sort = sortParam === 'desc' ? 'desc' : 'asc';
```